README.md

```diff
@@ -57,7 +57,7 @@ npm run build
 ```
 and that the selenium drivers are installed and up-to-date,
 ```
-./node_modules/gulp-protractor/node_modules/.bin/webdriver-manager update
+`npm bin`/webdriver-manager update
```

Comment:
with node 5.0.0 and npm 3.3.6 the file is still under `./node_modules/gulp-protractor/node_modules/.bin/webdriver-manager`. Which version do you use?